add support for react 17

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.

**Additional context**
Add any other context or screenshots about the feature request here.

Indeed. It works with React 17 flawlessly. 

However, due to the npm 7 behavior change on `peerDependencies`, we would have this error:

```
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.4.0" from overlayscrollbars-react@0.2.2
npm ERR! node_modules/overlayscrollbars-react
npm ERR!   dev overlayscrollbars-react@"^0.2.2" from the root project
```

@KingSora Would you mind to release a new version that relaxes the peerDependency?

--- a/packages/overlayscrollbars-react/package.json
+++ b/packages/overlayscrollbars-react/package.json
@@ -39,7 +39,7 @@
         "typescript": "^3.8.3"
     },
     "peerDependencies": {
-        "react": "^16.4.0",
+        "react": "^16.4.0 || ^17.0.0",
         "overlayscrollbars": "^1.10.0"
     },
     "scripts": {

